test_asn.json

```diff
@@ -0,0 +1 @@
+{"asn_rule": "Asn_Lv2Image", "asn_pool": "pool", "asn_type": "image2", "products": [{"name": "test_iris_subtract_bg_flat", "members": [{"expname": "temp_subarray_science.fits", "exptype": "science"}, {"expname": "/Users/cale/.astropy/cache/download/url/0b1ed4c136b7ef7db4e8202954e18fea/contents.fits", "exptype": "background"}]}]}
```

Comment:
need to fix paths

Reply:
I see the [jwst docs](https://jwst-pipeline.readthedocs.io/en/latest/jwst/associations/overview.html#what-are-associations) say the association files should live in the same directory as the .fits files and expnames should not contain the full directory. Since `test_asn.json` is created on the fly with data downloaded and stored in `~/.astropy/`, how do you recommend we resolve this?

Reply:
actually, `test_asn.json` is created by `test_image2`, so it should not be committed to the repo. Right?
This is just for testing, so even if we do not follow JWST best practices it is fine.

Reply:
Right, it should not committed. Should we add a flag to the `.gitignore`, or delete them in the test scripts after using them? And I agree violating the path rule is ok for this.

Reply:
the best would be to use `tmp_path`, so the file is automatically deleted after testing

Reply:
Fixed in [e50837c](https://github.com/oirlab/liger_iris_pipeline/pull/75/commits/e50837c99efa6c7a98d61b1efecb27e06d130a98)